articles/quickstart/native/ios-swift/interactive.md

```diff
@@ -190,9 +190,9 @@ Press the **Logout** button and verify that:
 :::
 
 :::checkpoint-failure
-Sorry about that. Here's a couple things to double check:
-- Did you enter the correct logout URL value?
-- Make sure the Auth0 Domain and Client ID are set correctly.
+If your application did not launch successfully:
```

Comment:
```suggestion
If login fails or returns errors:
```

Reply:
This one should be "If logout fails". Fixed it in  https://github.com/auth0/docs/pull/10102/commits/a71b9738052f489e27ac08f85ea89b2b6339c3f6.